K/Kissat/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Kissat"
+version = v"5.8.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/arminbiere/kissat.git", "abfa45fb782fa3b7c6e2eb6b939febe74d7270b7"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+
+cd $WORKSPACE/srcdir/kissat
+./configure -shared
+make
+mkdir -p ${libdir} ${bindir}
+cp build/kissat${exeext} ${bindir}/.
+cp build/libkissat.so "$libdir/libkissat.${dlext}"
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms(exclude=Sys.iswindows; experimental=true)
```

Comment:
Frankly I'm surprised this syntax works at all
```suggestion
platforms = supported_platforms(; exclude=Sys.iswindows, experimental=true)
```